rust/libnewsboat/src/fmtstrformatter/limited_string.rs

```diff
@@ -55,7 +45,7 @@ impl LimitedString {
         if let Some(limit) = self.max_length {
             let remaining_length = limit - self.length();
             self.content
-                .push_str(&utils::take_graphemes(s, remaining_length));
```

Comment:
Same as with `utils::graphemes_count`: is it used elsewhere? Can it be removed now?